chcemvediet/urls.py

```diff
@@ -13,9 +13,11 @@
 
 # Initializes the project
 from . import ready
+from .sitemaps import sitemaps
```

Comment:
Pls presun tento import k `from . import views`. Import `from . import ready` by mal byt oddeleny zvlast, pretoze to nie je import koli importu premennej z ineho suboru, ale koli inicializacii niecoho pred pustenim kodu v tomto moduli. Ked si vsimnes importnute `ready` sa nikde nevyuziva. Tak nech je ten import oddeleny od ostatnych. Takisto aj ten komentar sa tyka iba tohto importu a nie `sitemaps`

Na drslim to bolo v zlom poradi.

Reply:
:ok: